src/amber/router/params_parser.cr

```diff
@@ -0,0 +1,127 @@
+require "json"
```

Comment:
I don't like creating a whole new file with mostly the same contents of params.cr. This screws up git history.

Reply:
I renamed the file to something more meaningful.